docusaurus/docs/recipes/contentful.md

```diff
@@ -207,30 +214,29 @@ export const buildStore = (initialState, { query }) => {
     };
     
     // Add client as a thunk middleware
-    // This will give us access to the `client` object in our action functions as an argument
+    // This will give us access to the `client` object in our fetcher functions as an argument
     const middlewares = applyMiddleware(thunkMiddleware.withExtraArgument({ client }));
 
     return createStore(reducer, initialState, middlewares);
-    }
 }
 
 export default buildStore;
 ```
 
-Then, in your actions scripts, you can use the provided client to fetch the content that you need, like so:
+Then, in your fetcher scripts, you can use the provided client to fetch the content that you need, like so:
 
 ```js
-// usecases/actions.js
-// Create a fetch action that fetches data from Contentful
+// www/shared/utils/fetchers/useCases.js
```

Comment:
Same as above, please confirm the path is in conformity with our reduxe recipe.

Reply:
I see the problem here, I replaced this while unaware of the redux context, I'm reverting to what it was before